style.css

```diff
@@ -1,588 +1,983 @@
+.skills {
+  padding-left: 20px;
 }
 
-.about-text > hr {
-  padding-top: 100px;
-  border-top: none;
-  border-left: none;
-  border-right: none;
-  border-color: white;
+.lastpage {
+  background-color: #f7f7f7;
+  padding: 120px 12px 10px 12px;
 }
 
-.about-me .skill-set {
-  padding-top: 100px;
-  color: white;
+.lastpage-heading {
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
   text-align: center;
+  color: #172b4d;
+  margin-bottom: 10px;
 }
 
-.about-me .skill-set ul {
+.form {
+  width: 100%;
   display: flex;
-  justify-content: center;
+  align-items: center;
   flex-direction: column;
-  padding: 0;
+  padding: 19px 5px 24px 5px;
 }
 
-.skill-set li {
-  padding: 10px 0;
+.fullname {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  background-color: #f7f7f7;
+  color: #979493;
+  border: none;
+  border-bottom: 1px solid black;
+  padding-left: 20px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  margin-top: 54px;
 }
 
-.my-languages {
-  font-family: 'InaiMathi', sans-serif;
-  color: white;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
+.text-area {
+  height: 90px;
+  width: 100%;
+  color: black;
+  border: none;
+  border-bottom: 1px solid black;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  margin-top: 54px;
+  background-color: #fbf8f7;
+  padding-left: 20px;
 }
 
-.my-frameworks {
-  font-family: 'Inter', sans-serif;
-  color: white;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
+.button-end {
+  width: 131px;
+  margin: 30px 0 30px 0;
 }
 
-.my-skills {
-  font-family: 'Inter', sans-serif;
-  color: white;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
+.footer-line1 {
+  width: 100%;
+  margin-top: 70px;
 }
 
-.skill-item {
-  font-style: normal;
-  font-weight: normal;
-  font-size: 22px;
-  line-height: 43px;
-  list-style: none;
+.desktop-language-line {
+  display: none;
 }
 
-.vector {
-  margin: 1px;
-  width: 18px;
-  height: 18px;
-  border-radius: 150px;
+.last-page-footer {
+  margin-top: 34px;
+  display: flex;
+  justify-content: center;
+  padding: 24px;
+  width: 100%;
 }
 
-.contact {
-  background-color: #fff;
-  padding: 0 24px;
+.footer-line {
+  width: 100%;
+  height: 1px;
+  margin-bottom: 25px;
+}
+
+@media (min-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .humb {
+    display: none;
+  }
+
+  .menu-desktop-container {
+    display: flex;
+    flex-direction: row;
+    margin-right: 100px;
+  }
+
+  .menu-desktop {
+    font-family: 'Inter', sans-serif;
+    font-size: 15px;
+    text-align: center;
+    color: #fff;
+    text-decoration: none;
+    margin-left: 32px;
+  }
+
+  .logo {
+    margin-left: 100px;
+  }
+
+  .page1-content {
+    display: grid;
+    grid-template-columns: 5% 45% 50%;
+    height: fit-content;
+    padding-bottom: 20%;
+    width: 100%;
+    padding-top: 3%;
+  }
```

Comment:
[OPTIONAL] If you're going to have a fixed navbar, I recommend you bring it to the forefront of the page.

This is how your page is looking at the moment:

<img width="1440" alt="Screen Shot 2022-06-11 at 9 10 26" src="https://user-images.githubusercontent.com/82426682/173191366-6f8356aa-8719-4c81-a8dd-2550889de9a2.png">

The content of the section overlapping with the navbar doesn't make for UI/UX. Please consider fixing it.